### PR TITLE
Update filter options and fix bugs in item details selection

### PR DESCRIPTION
**Pull request by jaycellearianne**
https://github.com/user-attachments/assets/c10c486a-1204-4c07-a628-6a23681aaa4a

### CODE REVIEW

**Review by Dave00913 — APPROVED** [review]

File: components/home/pieces/PiecesTab.tsx [hunk 1]
```diff
@@ -60,7 +60,26 @@ const PiecesTab = () => {
     setSelectedFilters([]);
   };
 
-  const categoryNames = Object.keys(categoryTypes);
+  const getColorHexCode = (colorName: string): string | undefined => {
```
Comment (GangeiKan510) [inline components/home/pieces/PiecesTab.tsx]: pwede ka ka himo utility function yan then transfer mo sa `utils` directory ta. Medjo ga damo na ang lines ta sa amo ni nga file basi di na maintain kag budlay na ichange later on

File: components/home/pieces/PiecesTab.tsx [hunk 2]
```diff
@@ -75,6 +94,44 @@ const PiecesTab = () => {
     return false;
   };
 
+  const filterOptions = () => {
```
Comment (GangeiKan510) [inline components/home/pieces/PiecesTab.tsx:77]: same man diri

File: components/home/pieces/PiecesTab.tsx [hunk 3]
```diff
@@ -13,12 +13,12 @@ import Ionicons from "@expo/vector-icons/Ionicons";
 import FilterIcon from "../../../assets/icons/filter-icon.svg";
 import ClothingCard from "@/components/cards/ClothingCard";
 import { useUser } from "@/components/config/user-context";
-import { categoryTypes } from "@/components/constants/clothing-details/categories";
-import { occasion } from "@/components/constants/clothing-details/occasion";
+// import { categoryTypes } from "@/components/constants/clothing-details/categories";
```
Comment (GangeiKan510) [inline components/home/pieces/PiecesTab.tsx]: unused na ni and gin comment out?

**Review by GangeiKan510 — CHANGES_REQUESTED** [review]

File: components/home/pieces/PiecesTab.tsx [hunk 4]
```diff
@@ -75,6 +94,44 @@ const PiecesTab = () => {
...
+      season: new Set<string>(),
+      occasion: new Set<string>(),
+      category: new Set<string>(),
+      color: new Set<{ name: string; colorCode: string }>(),
+      material: new Set<string>(),
+      pattern: new Set<string>(),
+    };
+
+    user?.clothes.forEach((item) => {
+      item.season?.forEach((s) => filterOptions.season.add(s));
```
Comment (GangeiKan510) [inline components/home/pieces/PiecesTab.tsx]: please change to word gid like `season` instead of `s` lang

**Review by GangeiKan510 — APPROVED** [review]

**Pull request merged**